src/chrome/content/rules/Modern.IE.xml

```diff
@@ -44,6 +44,11 @@
 	<!--securecookie host="^remote\.modern\.ie$" name="^connect\.sid$" /-->
 
 	<securecookie host="^(?:\.devchannel|\.?remote|\.status|\.www)?\.modern\.ie$" name=".+" />
+	
+	<test url="https://www.modern.ie/" />
+	
+	<rule from="^https?://www\.modern\.ie/"
```

Comment:
We don't touch the https version.
Please also document the mismatch on the top.